go/vt/vtgate/vindexes/cfc_test.go

```diff
@@ -0,0 +1,452 @@
+package vindexes
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"vitess.io/vitess/go/sqltypes"
+	"vitess.io/vitess/go/vt/key"
+	"vitess.io/vitess/go/vt/proto/topodata"
+)
+
+func TestCFC(t *testing.T) {
+	t.Run("build cfc", func(t *testing.T) {
+		cases := []struct {
+			testName string
+			params   map[string]string
+			err      error
+			offsets  []int
+		}{
+			{
```

Comment:
nit: Reading this in my IDE that adds field names was much easier than on gh. I think having the fields names there explicitly, and removing most nil values should make this a little easier to scan

Reply:
field names added.